lib/src/classes/projection.dart

```diff
@@ -57,7 +57,7 @@ abstract class Projection {
 
   /// Named Projection: a [Projection] can be obtained from the [ProjectionStore] via it's name.
   /// null value will return if Projection not exists in store.
-  factory Projection(String code) {
+  static Projection? get(String code) {
```

Comment:
Breaking change, mention in CHANGELOG
https://dart.dev/null-safety/faq#how-should-i-migrate-a-factory-that-can-return-null